pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitLab , which, go-md2man, coreutils, substituteAll }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.4";
+  pname = "brillo";
+  src = fetchFromGitLab {
+    owner= "cameronnemo";
+    repo= "brillo";
+    rev= "v${version}";
+    sha256 = "0sadsqjxwvmq2jrvq8skvix3jg936k1qsv1x2hw8v9w6craghm99";
+  };
+  makeFlags = [ "PREFIX=$(out)" "AADIR=$(out)/etc/apparmor.d"];
+  nativeBuildInputs = [ go-md2man which ];
+  buildFlags = [ "dist" ];
+  installTargets = "install-dist";
+  patches = [
+  (substituteAll {
```

Comment:
The indentation, order of attributes and their spacing is somewhat less idiomatic but the code itself looks fine to me.

Reply:
I'm happy to go through and fix this in a little bit, is there a style guide/example of a package that does it well?

Reply:
I've moved round some stuff and I think it's slightly nicer now

Reply:
@alexarice you accidentally added a third space right before `meta`. Other than that I think it looks great!

Reply:
fixed, hopefully I haven't managed to break something else this time